src/accel_config.rs

```diff
@@ -0,0 +1,140 @@
+//! Pointer acceleration
+use crate::ffi;
+use crate::AccelProfile;
+use crate::DeviceConfigError;
+use crate::DeviceConfigResult;
+use input_sys::libinput_config_accel;
+
+/// A handle for configuration pointer acceleration.
+///
+/// @warning Unlike other structs pointer acceleration configuration is
+/// considered transient and <b>not</b> refcounted. Calling
+/// libinput_config_accel_destroy() <b>will</b> destroy the configuration.
```

Comment:
Looking at these copied docs, we need a `Drop` implementation for `AccelConfig` that calls `libinput_config_accel_destroy`.

Also please reword the docs to not mention ref-counting or destroy calls, as this is handled transparently.